### DISCUSSION TITLE
AlDente showing 1092h to charge

### DISCUSSION THREAD

**Discussion by UsamaKarim** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1327] [General]
1. Can you please explain about the warning icon and what does it mean?
2. Why AlDente is showing 1092h to charge the battery even the wattage is around 10W, The charger is 30W and it's connected to Anker USB C Hub.
![Screenshot 2024-09-17 at 7 10 21 PM](https://github.com/user-attachments/assets/a83cbc14-60eb-44fc-954b-fc146480a4af)

**Comment by MatthiasKerbl** [comment]
Hi @UsamaKarim,

Thank you for your feedback.

- The warning icon should appear when your MacBook is plugged into an external monitor but does not charge. However, it is not working as intended in your case, therefore, we will either try to fix it with the next update or remove it for now again.
- The "Time to full" number is coming from macOS which gets "confused" in some scenarios with AlDente. We are working on improving these scenarios with the next updates too.

Furthermore, there seem to be issues with some Anker USB C hubs. If you want to help us with the development, please share a debug file the next time this happens so that we can assess this issue better. You can find a guide on [How to generate and share a debug file](https://apphousekitchen.com/how-to-generate-and-share-a-debug-file/) on our blog.

Best,
Matthias

**Reply by UsamaKarim** [reply]
Sure.

**Reply by UsamaKarim** [reply]
[aldentepro-debug-2024-09-21-12:20:37.pasta.zip](https://github.com/user-attachments/files/17083812/aldentepro-debug-2024-09-21-12.20.37.pasta.zip)

Here is the debug file
